netci.groovy

```diff
@@ -2486,7 +2486,7 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
                     // ZIP up the generated CoreFX runtime and tests.
 
                     def dockerImage = getDockerImageName(architecture, os, true)
-                    def dockerCmd = "docker run -i --rm -v \${WORKSPACE}:\${WORKSPACE} -w \${WORKSPACE} -e ROOTFS_DIR=/crossrootfs/arm -e CAC_ROOTFS_DIR=/crossrootfs/x86 ${dockerImage} "
+                    def dockerCmd = "docker run -i --rm -v \${WORKSPACE}:\${WORKSPACE} -w \${WORKSPACE} -e ROOTFS_DIR=/crossrootfs/${architecture} -e CAC_ROOTFS_DIR=/crossrootfs/x86 ${dockerImage} "
```

Comment:
Just wondering if setting `CAC_ROOTFS_DIR` to some non-empty string value for `arm64` gonna break build?
In official build it is set to empty string for `arm64` https://github.com/dotnet/coreclr/blob/359f32036f0e72280c37343f7c3af1bbe0582828/buildpipeline/DotNet-CoreClr-Trusted-Linux-Crossbuild.json#L527-L530.

Reply:
I was hoping it would be ignored.  The alternative would be to special case it.

Reply:
It won't work

Reply:
Fixed